tauri-app/src/app/globals.css

```
@@ -255,4 +255,236 @@ body {
+  initial-value: 0deg;
+  inherits: false;
+}
+@keyframes cm-shortcut-spin {
+  to {
+    --cm-ring-angle: 360deg;
+  }
+}
+@layer utilities {
+  .cm-shortcut-listening {
+    --cm-shortcut-blue: #155dfc;
+    position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)